### ISSUE TITLE
ModuleNotFoundError: No module named 'cryptoadvance'

### ISSUE THREAD

**Issue by 4NobleTruths** [cryptoadvance/specter-desktop#494]
Installed with pip, as in instructions on Debian 10. No problems.

Tried to start with:
~~~
python3 -m cryptoadvance.specter server
~~~

Full error:
>/usr/bin/python3: Error while finding module specification for 'cryptoadvance.specter' (ModuleNotFoundError: No module named 'cryptoadvance')
~~~
pip3 list
~~~
shows module:
>cryptoadvance.specter 0.8.1

Binary runs.

**Comment by ben-kaufman** [comment]
Is it possible that you have multiple python3 versions installed on the same machine and `pip3` is using a different python version than the `python3` command is set to?

You can check by running this and comparing the outputs:
```
python3 --version
pip3 --version
```

**Comment by 4NobleTruths** [comment]
Yes, pip3 is already 3.8, but the highest available in the Debian repo is 3.7.3.
So, I probably had to upgrade pip3 manually to get something else to work, which always creates this same problem in python world, as you know.
So, I'll just drop it for now, until it's less beta. I can't get the binary to do what the instructions say anyway.
Thanks for your time.
